DevoxxClientMobile/src/main/java/com/devoxx/DevoxxApplication.java

```diff
@@ -221,6 +219,17 @@ public Button getSearchButton() {
         return navSearchButton;
     }
     
+    public MenuItem scanAsDifferentUser() {
+        final MenuItem scanAsDifferentUser = new MenuItem(DevoxxBundle.getString("OTN.BADGES.SCAN.DIFFERENT.USER"));
```

Comment:
After some sign in/logout attempts, the `Scan with Different User` button shows a wrong 
[menu item (click for image)](https://www.dropbox.com/s/ymq4df6pwzl6mwx/badges.png?dl=0).

Reply:
This bug is random and extremely difficult to re-create. The location of the MenuPopupView is correct. However, the size of the ScrollPane restricts Label from expanding. My guess would be that it has to do something with how we resize rootNode in `PopupView#layoutChildren()` but its extremely difficult to analyze without reproduction of the bug.